projects/configs/tag-operations/pet.yaml

```diff
@@ -76,10 +88,6 @@
   group: 0x0018
   element: 0x1076
   op: keep
-- name: Radiopharmaceutical Start DateTime
-  group: 0x0018
-  element: 0x1078
-  op: keep
```

Comment:
ah yep, we shouldn't be keeping this value. The attribute is [required by the spec](https://dicom.innolitics.com/ciods/enhanced-pet-image/enhanced-pet-isotope/00540016/00181078) but we can `replace` rather than `keep` the value